hack/deployer/runner/ocp.go

```diff
@@ -462,3 +462,29 @@ func (d *OCPDriver) baseDomain() string {
 	}
 	return baseDomain
 }
+
+func (d *OCPDriver) Cleanup(prefix string, olderThan time.Duration) ([]string, error) {
+	if err := d.authToGCP(); err != nil {
+		return nil, err
+	}
+	daysAgo := time.Now().Add(-olderThan)
+
+	params := d.bucketParams()
+	params["Date"] = daysAgo.Format(time.RFC3339)
+	params["E2EClusterNamePrefix"] = prefix
+	params["Region"] = d.plan.Ocp.Region
+
+	cmd := `gcloud compute instances list --verbosity error --zones={{.Region}}-a,{{.Region}}-b,{{.Region}}-c --filter="name~'^{{.E2EClusterNamePrefix}}-ocp.*-master.*' AND status=RUNNING" --format=json | jq -r --arg d "{{.Date}}" 'map(select(.creationTimestamp | . <= $d))|.[].name' | grep -o '{{.E2EClusterNamePrefix}}-ocp-[a-z]*-[0-9]*' | sort | uniq`
```

Comment:
Why do we filter on "masters" only? I wonder if we could hit a case where only the bootstrap vm has been created.
```suggestion
	cmd := `gcloud compute instances list --verbosity error --zones={{.Region}}-a,{{.Region}}-b,{{.Region}}-c --filter="name~'^{{.E2EClusterNamePrefix}}-ocp.*' AND status=RUNNING" --format=json | jq -r --arg d "{{.Date}}" 'map(select(.creationTimestamp | . <= $d))|.[].name' | grep -o '{{.E2EClusterNamePrefix}}-ocp-[a-z]*-[0-9]*' | sort | uniq`
```

Reply:
There's probably no great reason. My thought was that if the cluster is online, then the master has to be online, so search for signs of it. I'll update and we can modify in the future if needed.